### PR TITLE
Cell types olfactory

### PR DESCRIPTION
**Pull request by Peyman-N**
A forgotten pull request concerning cell types for olfactory system

### CODE REVIEW

File: instances/cellType/externalTuftedCell.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/externalTuftedCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": null,
```
Comment (lzehl) [inline instances/cellType/externalTuftedCell.jsonld:7]: Please always provide a one sentence definition.

File: instances/cellType/externalTuftedCell.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "External tufted cells have large somata and abundant cytoplasm containing stacks of Nissl material; their main dendrites are characterized by pale cytoplasm and a regular array of neurotubules. (PINCHING and POWELL 1971)",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107943",
+    "knowledgeSpaceLink": null,
+    "name": "external tufted cell",
+    "preferredOntologyIdentifier": null,
+    "synonym": null
```
Comment (lzehl) [inline instances/cellType/externalTuftedCell.jsonld:13]: please provide the synonyms as stated in the ontology

File: instances/cellType/externalTuftedCell.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/externalTuftedCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": null,
+    "description": "External tufted cells have large somata and abundant cytoplasm containing stacks of Nissl material; their main dendrites are characterized by pale cytoplasm and a regular array of neurotubules. (PINCHING and POWELL 1971)",
```
Comment (lzehl) [inline instances/cellType/externalTuftedCell.jsonld:8]: please make the citation a link if possible

Comment (lzehl) [reply instances/cellType/externalTuftedCell.jsonld:8]: please provide a full description or set to null and provide a good one sentence definition.

File: instances/cellType/granuleCellOlfactoryBulb.jsonld [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/granuleCellOlfactoryBulb",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "The main intrinsic neuron in the vertebrate olfactory bulb. It lacks an axon.",
```
Comment (lzehl) [inline instances/cellType/granuleCellOlfactoryBulb.jsonld:7]: Please provide this a one sentence at cite your source. Be clear if you are referring to the superclass of granule cells in the olfactory bulb or to the main or accessory granule cells.

Comment (lzehl) [reply instances/cellType/granuleCellOlfactoryBulb.jsonld:7]: adapt name if you a clear about this

File: instances/cellType/granuleCellOlfactoryBulb.jsonld [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "Each cell gives rise to short central dendrites and a single long apical dendrite that traverses the granule cell layer, pierces the mitral cell body layer, and branches and terminates within the external plexiform layer among the lateral dendrites of mitral and tufted cells. The dendrites receive synaptic input from mitral and tufted cell lateral dendrites, and have synaptic outputs on those dendrites through reciprocal dendrodendritic synapses.",
```
Comment (lzehl) [inline instances/cellType/granuleCellOlfactoryBulb.jsonld:8]: source is missing

File: instances/cellType/externalTuftedCell.jsonld [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107943",
+    "knowledgeSpaceLink": null,
```
Comment (lzehl) [inline instances/cellType/externalTuftedCell.jsonld:10]: KS link is missing but does exist: https://knowledge-space.org/wiki/NIFEXT:121#olfactory-bulb-main-tufted-cell

File: instances/cellType/granuleCellOlfactoryBulb.jsonld [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107930",
+    "knowledgeSpaceLink": null,
```
Comment (lzehl) [inline instances/cellType/granuleCellOlfactoryBulb.jsonld:10]: KS link is missing: at least for the two sub types (main or accessory) the links exist: 
https://knowledge-space.org/wiki/NLXCELL:20090401#olfactory-bulb-accessory-granule-cell
https://knowledge-space.org/wiki/NIFEXT:123#olfactory-bulb-main-granule-cell

File: instances/cellType/granuleCellOlfactoryBulb.jsonld [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    "name": "granule cell olfactory bulb",
+    "preferredOntologyIdentifier": null,
+    "synonym": null
```
Comment (lzehl) [inline instances/cellType/granuleCellOlfactoryBulb.jsonld:13]: there are synonyms for both sub types. please add them

File: instances/cellType/externalTuftedCell.jsonld [hunk 9]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/externalTuftedCell",
```
Comment (lzehl) [inline instances/cellType/externalTuftedCell.jsonld:5]: We need to investigate if we should call them external or superficial tufted cells. "Three subclasses of tufted cells are recognized based on location in the EPL: superficial, middle, and deep tufted cells (Cajal, R. S. Y., 1890; Pinching, A. J. and Powell, T. P., 1971a; Macrides, F. and Schneider, S. P., 1982; Orona, E. et al., 1984)"

Comment (lzehl) [reply instances/cellType/externalTuftedCell.jsonld:5]: https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/tufted-cell

File: instances/cellType/middleTuftedCell.jsonld [hunk 10]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/middleTuftedCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Principal neuron located in the outer third of the external plexiform layer of the olfactory bulb in the mammalian central nervous system.",
+    "description": "Each tufted cell is characterized by usually a single short primary dendrite that traverses the outer external plexiform layer and terminates within an olfactory glomerulus in a tuft of branches, where it receives the input from olfactory receptor neuron axon terminals. Differentiated from external tufted cells. Axons of the tufted cells transfer information to a number of areas in the brain, including the piriform cortex, entorhinal cortex, olfactory tubercle, and amygdala.",
```
Comment (lzehl) [inline instances/cellType/middleTuftedCell.jsonld:8]: citation of source is missing

File: instances/cellType/middleTuftedCell.jsonld [hunk 11]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/middleTuftedCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Principal neuron located in the outer third of the external plexiform layer of the olfactory bulb in the mammalian central nervous system.",
```
Comment (lzehl) [inline instances/cellType/middleTuftedCell.jsonld:7]: citation of source is missing

File: instances/cellType/middleTuftedCell.jsonld [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "Each tufted cell is characterized by usually a single short primary dendrite that traverses the outer external plexiform layer and terminates within an olfactory glomerulus in a tuft of branches, where it receives the input from olfactory receptor neuron axon terminals. Differentiated from external tufted cells. Axons of the tufted cells transfer information to a number of areas in the brain, including the piriform cortex, entorhinal cortex, olfactory tubercle, and amygdala.",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107935",
+    "knowledgeSpaceLink": null,
+    "name": "middle tufted cell",
+    "preferredOntologyIdentifier": null,
+    "synonym": null
```
Comment (lzehl) [inline instances/cellType/middleTuftedCell.jsonld:13]: synoyms are missing

File: instances/cellType/middleTuftedCell.jsonld [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/middleTuftedCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Principal neuron located in the outer third of the external plexiform layer of the olfactory bulb in the mammalian central nervous system.",
+    "description": "Each tufted cell is characterized by usually a single short primary dendrite that traverses the outer external plexiform layer and terminates within an olfactory glomerulus in a tuft of branches, where it receives the input from olfactory receptor neuron axon terminals. Differentiated from external tufted cells. Axons of the tufted cells transfer information to a number of areas in the brain, including the piriform cortex, entorhinal cortex, olfactory tubercle, and amygdala.",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107935",
+    "knowledgeSpaceLink": null,
+    "name": "middle tufted cell",
+    "preferredOntologyIdentifier": null,
```
Comment (lzehl) [inline instances/cellType/middleTuftedCell.jsonld:12]: we should discuss if this should be interlex if interlex is the defining ontological source

File: instances/cellType/tuftedCell.jsonld [hunk 14]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/tuftedCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Output cells similar to mitral cells but located more superficially in the external plexiform layer are called tufted cells (Shepherd 2004)",
```
Comment (lzehl) [inline instances/cellType/tuftedCell.jsonld:7]: please make the citation a link if possible

File: instances/cellType/externalTuftedCell.jsonld (hunk 9 above)
Comment (lzehl) [reply instances/cellType/externalTuftedCell.jsonld:5]: or is external tufted cell even a fourth sub type?

File: instances/cellType/mitralCell.jsonld [hunk 15]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/mitralCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Mitral cells are a neuronal cell type in the mammalian olfactory bulb, distinguished by the position of their somata located in an orderly row in the mitral cell layer of the bulb.",
```
Comment (lzehl) [inline instances/cellType/mitralCell.jsonld:7]: citation of source is missing

File: instances/cellType/mitralCell.jsonld [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "Principal neuron located in the olfactory bulb in the mammalian central nervous system. The cell bodies are arranged in a thin layer between the granule cell layer and the external plexiform layer. Each mitral cell is usually characterized in the mammal by a single primary dendrite that traverses the external plexiform layer and terminates within an olfactory glomerulus in a tuft of branches which receives input from the axons of olfactory receptor neurons. Axons of the mitral cells project to a number of areas in the brain, including the piriform cortex, entorhinal cortex, olfactory tubercle, and amygdala.",
```
Comment (lzehl) [inline instances/cellType/mitralCell.jsonld:8]: citation of source is missing

File: instances/cellType/mitralCell.jsonld [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107933",
+    "knowledgeSpaceLink": null,
```
Comment (lzehl) [inline instances/cellType/mitralCell.jsonld:10]: I think this is this one: https://knowledge-space.org/wiki/NIFEXT:120#olfactory-bulb-main-mitral-cell

File: instances/cellType/mitralCell.jsonld [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+    "name": "mitral cell",
+    "preferredOntologyIdentifier": null,
```
Comment (lzehl) [inline instances/cellType/mitralCell.jsonld:12]: interlex?

File: instances/cellType/mitralCell.jsonld [hunk 19]
```diff
@@ -0,0 +1,14 @@
...
+    "synonym": null
```
Comment (lzehl) [inline instances/cellType/mitralCell.jsonld:13]: missing

File: instances/cellType/periglomerularCell.jsonld [hunk 20]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/periglomerularCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Small intrinsic neuron in the glomerular layer of the olfactory bulb, with cell bodies surrounding the olfactory glomerulus.",
+    "description": " The cell body is 6-8 um in diameter, from which arises a short bushy dendrite that arborizes within a glomerulus, where it receives synaptic input from olfactory receptor cell axon terminals, and engages in dendrodendritic interactions with mitral/tufted cell dendrites. Occasionally, bitufted PG cells connected to two glomeruli are seen. The axon distributes laterally within the extraglomerular region, extending as far as 5-10 glomeruli away. Some PG cells appear to lack axons. Subtypes may be identified based on their biochemical constituents.",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107934",
+    "knowledgeSpaceLink": null,
+    "name": "periglomerular cell",
+    "preferredOntologyIdentifier": null,
+    "synonym": null
```
Comment (lzehl) [inline instances/cellType/periglomerularCell.jsonld:13]: missing

File: instances/cellType/periglomerularCell.jsonld [hunk 21]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/periglomerularCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Small intrinsic neuron in the glomerular layer of the olfactory bulb, with cell bodies surrounding the olfactory glomerulus.",
+    "description": " The cell body is 6-8 um in diameter, from which arises a short bushy dendrite that arborizes within a glomerulus, where it receives synaptic input from olfactory receptor cell axon terminals, and engages in dendrodendritic interactions with mitral/tufted cell dendrites. Occasionally, bitufted PG cells connected to two glomeruli are seen. The axon distributes laterally within the extraglomerular region, extending as far as 5-10 glomeruli away. Some PG cells appear to lack axons. Subtypes may be identified based on their biochemical constituents.",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107934",
+    "knowledgeSpaceLink": null,
```
Comment (lzehl) [inline instances/cellType/periglomerularCell.jsonld:10]: missing: https://knowledge-space.org/wiki/NLXCELL:091202#olfactory-bulb-main-periglomerular-cell

File: instances/cellType/periglomerularCell.jsonld [hunk 22]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/cellType/periglomerularCell",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+    "definition": "Small intrinsic neuron in the glomerular layer of the olfactory bulb, with cell bodies surrounding the olfactory glomerulus.",
```
Comment (lzehl) [inline instances/cellType/periglomerularCell.jsonld:7]: citation of source is missing

File: instances/cellType/periglomerularCell.jsonld [hunk 23]
```diff
@@ -0,0 +1,14 @@
...
+    "description": " The cell body is 6-8 um in diameter, from which arises a short bushy dendrite that arborizes within a glomerulus, where it receives synaptic input from olfactory receptor cell axon terminals, and engages in dendrodendritic interactions with mitral/tufted cell dendrites. Occasionally, bitufted PG cells connected to two glomeruli are seen. The axon distributes laterally within the extraglomerular region, extending as far as 5-10 glomeruli away. Some PG cells appear to lack axons. Subtypes may be identified based on their biochemical constituents.",
```
Comment (lzehl) [inline instances/cellType/periglomerularCell.jsonld:8]: citation of source is missing

File: instances/cellType/periglomerularCell.jsonld [hunk 24]
```diff
@@ -0,0 +1,14 @@
...
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0107934",
+    "knowledgeSpaceLink": null,
+    "name": "periglomerular cell",
+    "preferredOntologyIdentifier": null,
```
Comment (lzehl) [inline instances/cellType/periglomerularCell.jsonld:12]: interlex?

File: instances/cellType/tuftedCell.jsonld (hunk 14 above)
Comment (lzehl) [reply instances/cellType/tuftedCell.jsonld:7]: this definition should be improved

Comment (lzehl) [reply instances/cellType/tuftedCell.jsonld:7]: e.g.:

"Tufted cells can be found in the external plexiform layer (EPL) and glomerular layer (GL) of the olfactory bulb and should be further subdivided into the superficial, middle and deep tufted cells of the EPL and the external tufted cells of the GL (Cajal, R. S. Y., 1890; Pinching, A. J. and Powell, T. P., 1971; Macrides, F. and Schneider, S. P., 1982; Orona, E. et al., 1984; Ennis, M., et al., 2007)." 

Cajal, R. S. Y., 1890: ?
Pinching, A. J. and Powell, T. P., 1971: https://doi.org/10.1242/jcs.9.2.305
Macrides, F. and Schneider, S. P., 1982: https://doi.org/10.1002/cne.902080410
Orona, E. et al., 1984: https://doi.org/10.1002/cne.902260305
Ennis, M., et al., 2007: https://doi.org/10.1007/978-0-387-30374-1_6

File: instances/cellType/externalTuftedCell.jsonld (hunk 9 above)
Comment (lzehl) [reply instances/cellType/externalTuftedCell.jsonld:5]: clarified this question I think in my comment for the superclass "tufted cells" (which means that I think two subtypes are missing from this PR which we should add as well)

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
replaced by https://github.com/openMetadataInitiative/openMINDS_instances/pull/87

**Pull request closed without merging**